tests/integration/utils.py

```diff
@@ -275,3 +291,14 @@ def assert_content_manifest(client, request_id, image_contents):
     response_data = content_manifest_response.data
     assert_content_manifest_schema(response_data)
     assert image_contents == content_manifest_response.data["image_contents"]
+
+
+def assert_properly_completed_response(completed_response):
```

Comment:
I think we can use it in other tests with another commit. But it can be a new PR.

Reply:
Yes, I am planning to do so.